CONVENTIONS.md

```diff
@@ -0,0 +1,75 @@
+MCP Naming Conventions
+======================
+
+In order to guarantee a consistent style and theme across MCP names, the following conventions
+should be adhered to when mapping new things or proposing replacements for existing mappings.
+Please refer to this document whenever you do not know how to format or structure a name.
+
+Getting Started
+---------------
+ - **If you do not know what to name something then *do not name it*. Let someone else do so.**
+ - Names should reflect intent, *not* implementation. Check usages rather than just looking at the definition.
+ - Names should be concise and memorable. If information can be inferred from context (type, parameters etc.),
+   do not include it in the name. This especially applies to fields and methods in a class, where you do not
+   need to have the defining class's name in the member's name.
+ - Prefer clarification in the documentation comment over verbose names. Conversely, do not give fields,
+   methods, or method parameters useless comments.
```

Comment:
Defining what "useless" is is somewhat important.  Describing what it does is (IMO) always useful even if redundant, except in perhaps the most obvious of cases.  On the other hand, a description of mechanics for an individual thing is generally not useful (e.g. for fields in `Blocks` or `Enchantments`), although I'd say it still is useful for mechanics that are fairly technical in nature/examples of where something might be used.

Reply:
I've switched to `needless` and added a few exemplary details that should go into the comment.